ontologies/vocabs.ttl

```diff
@@ -1168,8 +1154,10 @@ Tag for the government public services industry sector"""^^xsd:string ;
 
 gist:_IndustrySector_government-Regional
 	a gist:IndustrySector ;
-	rdfs:label "Government Regional"^^xsd:string ;
-	skos:altLabel "Government-Regional"^^xsd:string ;
+	skos:altLabel
+		"Government Regional"^^xsd:string ,
+		"Government-Regional"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?